File: src/main/java/com/gestion/absence/entities/ResponseSeance.java
package com.gestion.absence.entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ResponseSeance {
    private Long id;
    private Date dateDebut;
    private Date dateFin;
    private String room;
    private String subject;
    private int numberOfPresent;
    private int numberOfAbsence;
    private int numberOfRetard;
    private int numberOfAppelNonFaite;
}

File: ng-project/src/app/shared/modal/response-seance.ts
export interface ResponseSeance {
    id: number;
    dateDebut: Date;
    dateFin: Date;
    room: string;
    subject: string;
    numberOfPresent: number;
    numberOfAbsence: number;
    numberOfRetard: number;
    numberOfAppelNonFaite: number;
}
